yt_dlp/extractor/mocha.py

```diff
@@ -30,15 +30,14 @@ def _real_extract(self, url):
             'http://apivideo.mocha.com.vn:8081/onMediaBackendBiz/mochavideo/getVideoDetail',
             video_slug, query={'url': f'{url}', 'token': ''})
 
-        video_url = traverse_obj(json_data, ('data', 'videoDetail', 'list_resolution'))
-        video_url.append(traverse_obj(json_data, ('data', 'videoDetail', 'original_path')))
+        video_url = traverse_obj(json_data, ('data', 'videoDetail', ('list_resolution', 'original_path')))
 
         formats, subtitles = [], {}
         for video in video_url:
             if isinstance(video, str):
                 formats.extend([{'url': video, 'ext': 'mp4'}])
             else:
-                vid_url, subs = self._extract_m3u8_formats_and_subtitles(video['video_path'], video_slug, ext='mp4')
+                vid_url, subs = self._extract_m3u8_formats_and_subtitles(video[0]['video_path'], video_slug, ext='mp4')
```

Comment:
Is it fine to set `[0]` at this ?

Reply:
since u have another branch which can give formats, this will make the extractor fatal when it's not needed. Use traverse_obj here.

Reply:
Fixed